domains/service/src/lib.rs

```diff
@@ -24,3 +47,336 @@ pub struct DomainConfiguration<AccountId> {
     pub service_config: ServiceConfiguration,
     pub maybe_relayer_id: Option<AccountId>,
 }
+
+/// Build the network service, the network status sinks and an RPC sender.
+///
+/// Port from `sc_service::build_network` mostly the same with block sync disabled.
+#[allow(clippy::type_complexity)]
+pub fn build_network<TBl, TExPool, TImpQu, TCl>(
```

Comment:
This function TBH feels more like a penalty and overhead. Not a blocker but can't we shave off sync related stuff like wrap sync from these ?